packages/babel-register/src/node.js

```diff
@@ -6,6 +6,7 @@ import * as registerCache from "./cache";
 import extend from "lodash/object/extend";
 import * as babel from "babel-core";
 import each from "lodash/collection/each";
```

Comment:
travis just has a linting error after removing the each - just remove this line.